unimport/scan.py

```diff
@@ -24,7 +25,7 @@ class Scanner(ast.NodeVisitor):
     """To detect unused import using ast"""
 
     ignore_imports = ["__future__"]
-    skip_comments = ["#unimport:skip", "# unimport:skip"]
+    skip_comments_regex = "# unimport:skip|# noqa"
```

Comment:
Perhaps something a little more forgiving whitespace-wise, such as `#\s*(unimport:skip|noqa)`?

Reply:
Yes this is a great idea. thanks.

https://github.com/hakancelik96/unimport/commit/96eabd6dcacfd84c29aa94f0426073ba039dce5e